scripts/generate_reference_docs.py

```diff
@@ -0,0 +1,12 @@
+from pathlib import Path
```

Comment:
Please move this reference scripts into separate directory, they should not be a part of the main codebase